README.md

```diff
@@ -98,10 +98,20 @@ An example to get source codes and compile under **Linux**. You'd need at least
 $ git clone https://github.com/aregtech/areg-sdk.git
 $ cd areg-sdk
 ```
+
 ```bash
-# Step 2: Compile sources from terminal by calling: make [all] [framework] [examples]
-$ make all
+# Step 2: Create a subdirectory for CMake cache files and change directory to it.
+$ mkdir build && cd build
+
+# Step 3: Initialize CMake cache and build systems configuration.
+# Enable examples compilation by using BUILD_TESTS flag.
+$ cmake .. -DBUILD_TESTS=ON
```

Comment:
- Can we rename `BUILD_EXAMPLES`? Because there will be unit tests and this flag we can use for that
- By default, we can have `build all` (or `BUILD_ALL`, this should be set without option)
- `build framework` -- build only framework.
- `build examples` -- build framework + examples
- `build tests` -- build framework + unit tests (planned to integrate in next version)